### PR TITLE
Updated speaker card lists. Increased font size and bolded the first …

### PR DESCRIPTION
**Pull request by vgvbach95**
Updated speaker card lists. Increased font size and bolded the first letter. Shorttened a few bios

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -419,9 +419,9 @@ <h3 class="schedule-day-title">Saturday, October 1st</h3>
...
-								Formerly, Basil was the founder, President and CEO of Timetra, a start-up focused on service routers for IP/MPLS networks, which was acquired by Alcatel in 2003 and merged with the MSWAN and NSM business lines to form the IP Division. Prior to TiMetra, Basil held executive positions in Silicon Valley start-ups as well as large companies. Basil served as a member of the boards of directors of multiple Silicon Valley based companies including 2Wire, Omneon Video Networks, CiiNow and WiChorus.
-								In 1984, Basil Alwan graduated from the University of Illinois at Champaign-Urbana with a Bachelor’s degree in Computer Engineering.
+								<b>B</b>asil was perviously President of Alcatel-Lucent’s IP Division (IPD) with overall responsibility for the company’s portfolio of IP/MPLS and Multiservice switches and routers, and Network and Service management. He was also responsible for the portfolio strategy for the Carrier Group.
```
Comment (YashoSharma) [inline index.html]: There appears to be a typo on the third word "previously"

Comment (vgvbach95) [inline index.html]: Thanks for catching that.

**Comment by sskhandek** [comment]
screen shot 2016-09-22 at 2 02 34 pm

This gets cut off

screen shot 2016-09-22 at 2 02 56 pm

**Comment by vgvbach95** [comment]
That's what I was afraid of. Let's see what else I can trim.

**Pull request merged**